templates/workspaces/base/terraform/providers.tf

```diff
@@ -8,9 +8,9 @@ terraform {
       source  = "hashicorp/azuread"
       version = "=2.20.0"
     }
-    random = {
-      source  = "hashicorp/random"
-      version = "=3.1.2"
+    null = {
```

Comment:
Not sure about this change, is that came from the linter?

Reply:
Thanks. switched my mistake instead of adding.

Reply:
Actually, it wasn't by mistake. The updated version is correct.

Reply:
Ok, thanks